.github/workflows/ci.yml

```diff
@@ -9,12 +9,12 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        python-version: [2.7, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9, "3.10", pypy2, pypy3]
+        python-version: [2.7, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9, "3.10", 3.11, pypy2.7, pypy3.9]
 
     steps:
-    - uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5 # v2
+    - uses: actions/checkout@v3
```

Comment:
Could you expand these updates to the corresponding commit IDs please?

(cf. <https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies>, which links to <https://app.stepsecurity.io/> for a tool)

Reply:
Amended [dd63f3e](https://github.com/daviddrysdale/python-phonenumbers/pull/265/commits/dd63f3e471fe9d6d3030b1d9e0f607e9086a31d7) :)